pennylane/drawer/draw.py

```diff
@@ -306,8 +306,9 @@ def draw_mpl(
             Default ``None`` will omit parameters from operation labels.
         style (str): visual style of plot. Valid strings are ``{'black_white', 'black_white_dark', 'sketch',
             'sketch_dark', 'solarized_light', 'solarized_dark', 'default'}``. If no style is specified, the
-            ``'black_white'`` style will be used. Setting style does not modify matplotlib global plotting settings.
-            If ``None``, the current matplotlib settings will be used.
+            the current matplotlib settings will be used, and the initial default is 'black_white'. If you would
+            like to use your environment's current rcParams, set `style` to "rcParams".
+            Setting style does not modify matplotlib global plotting settings.

```

Comment:
:100:

Reply:
So is this how things will work?
- Any MPL drawings that are unrelated to PennyLane have their default styles unchanged
- The default for PL-side MPL drawings is [black and white](https://docs.pennylane.ai/en/stable/_images/black_white_style1.png). Users do not need to set `qml.drawer.use_style(style)` to get this style.
- If a user sets `qml.drawer.use_style(style)`, then it is the default for all PL-side MPL circuit plots, but still doesn't affect MPL plots in general
- `style` in `draw_mpl` overrides the default for that plot

Reply:
correct!

Reply:
I'm fine to remove it, but it was already there before